libc/utils/hdrgen/hdrgen/header.py

```diff
@@ -50,9 +57,7 @@
 HEADER_TEMPLATE = """\
 //===-- {library} header <{header}> --===//
```

Comment:
Unrelated, but since you're changing it the file header format for llvm no longer has the file name bits by default. 

It's fine for libc to make a different choice, but I just wanted to point out that it's not required per the coding standards anymore. 

https://llvm.org/docs/CodingStandards.html#file-headers

Reply:
This change is not meant to affect the output format.  We can address those issues separately.